node/core/dispute-coordinator/src/lib.rs

```diff
@@ -71,10 +71,27 @@ const ACTIVE_DURATION_SECS: Timestamp = 180;
 /// Timestamp based on the 1 Jan 1970 UNIX base, which is persistent across node restarts and OS reboots.
 type Timestamp = u64;
 
+#[derive(Eq, PartialEq)]
+enum Recovery {
```

Comment:
nit: `Recovery` doesn't seem like the right name, since it's not about recovery but participation on startup.